Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/repair.h

```diff
@@ -186,6 +184,289 @@ degenerate_faces(const TriangleMesh& tm, OutputIterator out)
   return degenerate_faces(tm, get(vertex_point, tm), Kernel(), out);
 }
 
+/// \ingroup PMP_repairing_grp
+/// checks whether a vertex is non-manifold.
+///
+/// @tparam TriangleMesh a model of `FaceListGraph` and `MutableFaceGraph`
+///
+/// @param v the vertex to check whether is degenerate
+/// @param tm the triangulated surface mesh upon evaluation
+///
+/// \return true if the vertrex is non-manifold
+template <typename TriangleMesh>
```

Comment:
TriangleMesh ->PolygonMesh. The algorithm is not only working on triangle meshes